kwave/options/simulation_execution_options.py

```diff
@@ -34,6 +34,9 @@ class SimulationExecutionOptions:
     system_call: Optional[str] = None
     verbose_level: int = 0
 
+    # show simulation log
+    show_sim_log: bool = True
```

Comment:
This variable is not used, what's the reason for introducing it?

Reply:
used in line 31 of executor.py to pipe the stdout to DEVNULL

Reply:
Ahh okay thanks!